pdx-extract/tests/test_main.py

```diff
@@ -0,0 +1,280 @@
+import asyncio
+import unittest.mock as mock
+
+import asynctest
+import pytest
+
+from app import main
+from tests import data
+from tests import utils
+
+
+@mock.patch("requests.post")
+def test_authenticate_current_session_returns_ok(mock_requests):
+    import requests
+
+    term = {"code": "201904", "description": "Fall 2019 Quarter"}
+    unique_session_id = "abcdef1234567890"
+    cookies = {"JSESSIONID": "CF6813D3F9BFD1ABEEEF47E2FB094926"}
+    response = requests.Response()
+    response.status_code = 200
+    mock_requests.return_value = response
+
+    res = main.authenticate_current_session(term, unique_session_id, cookies)
+
+    assert res.ok is True
+
+
+@mock.patch("requests.post")
+def test_authenticate_current_session_returns_not_ok_if_response_fails(mock_requests):
+    import requests
+
+    term = {"code": "201904", "description": "Fall 2019 Quarter"}
+    unique_session_id = "abcdef1234567890"
+    cookies = {"JSESSIONID": "CF6813D3F9BFD1ABEEEF47E2FB094926"}
+    response = requests.Response()
+    response.status_code = 400
```

Comment:
Nitpicky, but this is a magic number. Do one of the libraries you're using have these somewhere you can reference?

Reply:
Sweet, this made me do some digging and discover that the library I'm using does support this status codes. Thanks!